### ISSUE TITLE
Feral human's throwing still seems wrong

### ISSUE THREAD

**Issue by Trioct** [CleverRaven/Cataclysm-DDA#58645]
### Describe the bug

With a character at 0 dex/0 dodge, ferals human rock throws cannot miss, and it almost always hits the torso.
Here are some statistics on two runs of 30 feral rock throws (aka 5 feral humans, sequentially so they don't hit themselves) at max range with a naked character.

Commit 66f9ecfb9696b47fb7a95337061aefe97f11d2d7, #51465
```
Avg damage per throw: 9.6
One feral = 35 torso damage/57.6 total damage in rocks
Torso damage: 175 (if 84 torso hp, then 2.083 deaths, 2.4 feral humans worth of rocks per death)
Total damage: 288

+----------+------+------------+
| Location | Hits | Percentage |
+----------+------+------------+
| Head     | 0    | 0%         |
| Torso    | 18   | 60%        |
| Arms     | 10   | 33.333333% |
| Legs     | 2    | 6.6666666% |
| Total    | 30   | 100%       |
+----------+------+------------+
```

Master branch
```
Avg damage per throw: 8.76
One feral = 49.2 torso damage/52.6 total damage in rocks
Torso damage: 246 (if 84 torso hp, then 2.93 deaths, 1.707 feral humans worth of rocks per death)
Total damage: 263

+----------+------+------------+
| Location | Hits | Percentage |
+----------+------+------------+
| Head     | 0    | 0%         |
| Torso    | 28   | 93.333333% |
| Arms     | 1    | 3.3333333% |
| Legs     | 1    | 3.3333333% |
| Total    | 30   | 100%       |
+----------+------+------------+
```

These aren't particularly long runs, but I think they're in the ballpark. Test it yourself.
Also notice that after #51465, I still was unable to be missed at 0 dodge. Reason being is their skill is too high to miss at their max range. Maybe 2 throwing skill and 10 rocks is a better way to go?
Just want to emphasize that if you end up near 2 feral humans,  low dodge/high encumbrance, full torso hp, even decently clothed, you have a very fair chance of dying within their 12 rocks if you don't kill them or get away. Especially brutal considering they're common day 1 and can open doors.
Why the torso seems more selected now? Not sure. Probably needs further testing, but I did a larger sample before this and the torso is probably hit ~90% of the time.

### Steps to reproduce

1. Get naked (or don't, normal, or even decent clothing doesn't really help at all)
2. Get dodge to 0 or set dex to 0
3. Find 2 roofs with a gap a feral human can throw over
4. Spawn 2 feral humans on the other roof and stand at their max range
5. Realize you forgot your torso at home and die

### Expected behavior

Feral humans should miss and not essentially only hit your torso.

### Versions and configuration

- OS: Linux
    - OS Version: LSB Version: n/a; Distributor ID: Gentoo; Description: Gentoo Base System release 2.7; Release: 2.7; Codename: n/a; 
- Game Version: 0.F-9279-gbcd8b3b094-dirty [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions]
]

**Comment by mkrutov** [comment]
/confirmed 

My feel is also that the rock-throwing is quite overpowered for ferals, and lowering their skill in json does not decrease their hit chances: they almost always hit (just deal less damage if you lower their skill).

**Comment by anoobindisguise** [comment]
Another aspect of this is that rocks are atypical projectiles, they are large and slow moving, it is entirely reasonable that they could be deflected with specific weapons (namely a baseball bat).

Yet another issue with ferals in general (firearm ferals too) is that they can very frequently hit you with projectiles from behind a wall of zombies. I frequently find that I am fighting a mob of zeds thru a window, and then a feral shows up and starts pelting me with rocks from behind 2-3 zombies climbing through the window. They only very very rarely actually hit their allies when throwing rocks this way. Even worse with firearm ferals since they're so lethal and it's even less logical that they can hit you in such a way.

**Comment by Gotdamnmiracle** [comment]
I think a good fix would be a rock throwing variant of feral, rather than several variants all deciding the carry 3 or 4 rocks on the off chance they need to huck something at someone.

**Comment by anothersimulacrum** [comment]
Can you compare this with an equivalent player's rock throwing?

**Comment by anothersimulacrum** [comment]
They do not have infinite ammo.
https://github.com/CleverRaven/Cataclysm-DDA/blob/e34eeaacc7a51fe8bb1798d662bee648749a70c1/data/json/monsters/feral_humans.json#L31
